### PR TITLE
BCMOHAD-14077||Added New Formula Fields

### PR DESCRIPTION
**Pull request by ahshaik1**
Deployment Tracker:
Fields :
1. Object : S1_HC_2023__c Field : COD_Q40A__c.field-meta.xml S1_HC_2023__c
2. Object : S1_HC_2023__c Field : COD_Q40B__c.field-meta.xml S1_HC_2023__c
3. Object : S1_HC_2023__c Field : COD_Q40C__c.field-meta.xml S1_HC_2023__c
4. Object : S1_HC_2023__c Field : COD_Q40D__c.field-meta.xml S1_HC_2023__c
5. Object : S1_HC_2023__c Field : COD_Q40E__c.field-meta.xml S1_HC_2023__c
6. Object : S1_HC_2023__c Field : COD_Q40F__c.field-meta.xml S1_HC_2023__c
7. Object : S1_HC_2023__c Field : COD_Q40G__c.field-meta.xml S1_HC_2023__c
8. Object : S1_HC_2023__c Field : COD_Q40H__c.field-meta.xml S1_HC_2023__c
9. Object : S1_HC_2023__c Field : COD_Q40I__c.field-meta.xml S1_HC_2023__c
10. Object : S1_HC_2023__c Field : COD_Q40J__c.field-meta.xml S1_HC_2023__c
11. Object:1_HC_2023__c Field:COD_S40I__c.field-meta.xml
Permission Sets:
1.MAiD_Analyst.permissionset-meta.xml
2.MAiD_Manager.permissionset-meta.xml
3.MAiD_Reports_Dashboards.permissionset-meta.xml
4.MAiD_System_Administrator.permissionset-meta.xml
Report:
MAiDCaseManagementReports/S7_2023_HC_ReportDied_Prior_XPB.report-meta.xml

### CODE REVIEW

**Comment by cgiSunilM** [comment]
@ahshaik1 Also i am not able to see COD_S40I__c.field-meta.xml in the deployment tracker description. Please update the same. Thanks!

**Comment by ahshaik1** [comment]
> @ahshaik1 Also i am not able to see COD_S40I__c.field-meta.xml in the deployment tracker description. Please update the same. Thanks!

@cgiSunilM : I have added it to the description in the tracker

File: MAiD - Case Management App/force-app/main/default/objects/S1_HC_2023__c/fields/COD_S40I__c.field-meta.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>COD_S40I__c</fullName>
+    <externalId>false</externalId>
+    <formula>Form_1634__r.Other_natural_death_reason__c</formula>
+    <formulaTreatBlanksAs>BlankAsZero</formulaTreatBlanksAs>
```
[the remark this answers is not in the document]
Comment (cgiSunilM) [reply MAiD - Case Management App/force-app/main/default/objects/S1_HC_2023__c/fields/COD_S40I__c.field-meta.xml]: Thanks for resolving.

**Review by cgiSunilM — APPROVED** [review]
LGTM!

**Pull request merged**